quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/integration/QuarkusPlatform.java

```diff
@@ -35,6 +35,13 @@
 
     private static final Logger log = Logger.getLogger(QuarkusPlatform.class);
 
+    Runnable startupHook;
```

Comment:
I won't do code formatting until we have something in place that we can share. See https://issues.redhat.com/browse/KEYCLOAK-16322.